src/utils/api/apiAuth.ts

```diff
@@ -8,13 +11,35 @@ export const authorizedJWTs = [
 ];
 
 export function isTokenValid(token: string) {
-	if (!token) {
+	let tokenCopy = token;
+
+	if (tokenCopy.startsWith('Bearer ')) {
+		// take out the "Bearer " part
+		tokenCopy = tokenCopy.substring(7);
+	}
+
+	if (!tokenCopy) {
```

Comment:
To avoid the mutability problem, you could probably do the following instead:

```suggestion
	if (!token.startsWith('Bearer ')) {
		return false;
	}

	const tokenValue = token.substring(7);
	if (!tokenValue) {
```

Alternatively, call the original function argument "authHeader" because that's what it is and you're just parsing it further into the token.